README.md: Direct link to SC 3.1.5

cstrobbe opened the pull request.

cstrobbe committed:
Direct link to SC 3.1.5

xi commented on README.md:
```diff
@@ -22,7 +22,7 @@ This could be useful for a lot of people:
 -	second language learners
 
-Among other things, it can be used to conform with [WCAG][2] 3.1.5:
+Among other things, it can be used to conform with [Success Criterion 3.1.5 in WCAG 2.0][2]:
```
This should be wrapped to stay within 80 character per line.

xi commented on README.md:
Is there any harm in linking to <https://www.w3.org/TR/WCAG20/#meaning-supplements>? I do not expect breaking changes there.

cstrobbe replied:
Linking directly to https://www.w3.org/TR/WCAG20/#meaning-supplements should be fine. Any changes to the SC would end up in WCAG 2.1.

xi reviewed (changes requested):
Thanks for the contribution! I have just some stylistic remark.

cstrobbe committed:
Link to undated WCAG URL.

The link now goes to https://www.w3.org/TR/WCAG20/#meaning-supplements instead of http://www.w3.org/TR/2008/REC-WCAG20-20081211/#meaning-supplements.

cstrobbe committed:
line break to keep line length less than 80 chars

xi commented:
Thanks
